Fix include order for workaround in interfaces/gz-msgs/patches/patch-ab
This fixes major(3)/minor(3) conflicts

--- src/FuelClient.cc~	2024-09-26 21:09:24.000000000 +0200
+++ src/FuelClient.cc	2025-03-02 15:31:00.086807743 +0100
@@ -37,8 +37,8 @@
 #include <gz/common/URI.hh>
 #include <gz/common/Util.hh>
 
-#include <gz/msgs/fuel_metadata.pb.h>
 #include <gz/msgs/Utility.hh>
+#include <gz/msgs/fuel_metadata.pb.h>
 
 #include "gz/fuel_tools/ClientConfig.hh"
 #include "gz/fuel_tools/CollectionIdentifier.hh"
--- src/Interface.cc~	2024-09-26 21:09:24.000000000 +0200
+++ src/Interface.cc	2025-03-02 15:31:57.727411106 +0100
@@ -24,9 +24,9 @@
 #pragma warning(pop)
 #endif
 
-#include <gz/msgs/fuel_metadata.pb.h>
 
 #include <gz/msgs/Utility.hh>
+#include <gz/msgs/fuel_metadata.pb.h>
 #include "gz/common/Console.hh"
 #include "gz/fuel_tools/Interface.hh"
 #include "gz/fuel_tools/WorldIdentifier.hh"
--- src/gz.cc~	2024-09-26 21:09:24.000000000 +0200
+++ src/gz.cc	2025-03-02 15:32:30.001274953 +0100
@@ -32,8 +32,8 @@
 #include <csignal>
 #include <exception>
 
-#include <gz/msgs/fuel_metadata.pb.h>
 #include <gz/msgs/Utility.hh>
+#include <gz/msgs/fuel_metadata.pb.h>
 
 #include <algorithm>
 #include <chrono>
